docs/features/note-templates.md

```diff
@@ -65,12 +67,12 @@ By using the `FOAM_DATE_` versions of the variables, the correct relative date w
 For example, given this daily note template (`.foam/templates/daily-note.md`):
 
 ```markdown
-# $FOAM_DATE_YEAR-$FOAM_DATE_MONTH-$FOAM_DATE_DATE
+# $FOAM_DATE_YEAR-$FOAM_DATE_MONTH-\$FOAM_DATE_DATE
```

Comment:
why this?

Reply:
Oops, this must have been formatted automatically. I'll remove this backslash.